### DISCUSSION TITLE
Some Java/Vanilla Tweaks parity options & Some new options

### DISCUSSION THREAD

**Discussion by Apoll0Music** [BedrockTweaks/Files#87] [Ideas]
Obviously I'm not quite sure if all or any of these are possible since I don't know a thing about coding, but if they are i would love to see some of these Java features added for bedrock. Also with the ability for addons now, I think some of these could be possible. 

Java/Vanilla Tweaks
- Utility Durabillity Stats ([Insert Number]/[Insert Number])
- Stat Tracking (Honestly I wouldn't know how to do this, but it's probably been done)
- Twinkling Stars
- World of Color Enchantment Glints, Hearts(And LGBTQ+ Hearts) & Elytra
- Variated Hunger Bars
- Transparent UI
- Dont Eat The Bowl (Remove eating effects from stews)

New Options
- Crosshair Opacity
- Old Rose Bush Texture
- Pocket Edition Zombie Pigman
- Grey Tabby Cat
- Always have music playing (Similar to Legacy Console)
- Ambianet sounds (Birds chirping, wind, etc.)

I think that is all of the things I can think of, as I said I'm not quite sure if all or any of these are possible and if not that's perfectly ok.

**Comment by SquatchHunter** [comment]
I would say all those packs are possible, given enough time.

weighing in my 2 cents on some design decisions for the addons specifically.
- Durability (and consequently, Durability Ping) would be possible, ~~but may need to use the "Lore" section of an item since I do not believe from runtime we can edit the item hover popup~~ nop its possible.
- Stat Tracking - very possible but very involved, through the use of a lot of actor_properties or scoreboards, either way, a lot to track if you were to track "every trackable event in game"
- Always Music - since there is a pack out there that not only allows for custom music but also adds a "music player" to the game, its possible but also a rather involved pack

and now for my suggestion of a new pack to add
**Parity**
- Death Counter - adds a scoreboard that tracks deaths, this would fit the parity category as scoreboards in Java edition can be of type "deaths" which does not exist in Bedrock currently.

_I thought I had some more in mind, but alas I have forgetten._

**Reply by AtlasOfEden** [reply]
Durability please!
